README.rst

```diff
@@ -119,7 +125,7 @@ In this repository we provide a `Dockerfile <Dockerfile>`__ to compile Octopus
 13.0 and `Dockerfile-develop <Dockerfile-develop>`__ to compile the ``develop``
```

Comment:
Here its should also be 14.0
```suggestion
14.0 and `Dockerfile-develop <Dockerfile-develop>`__ to compile the ``develop``
```
Side note the Dockerfile-develop related content needs to go. issue @ https://github.com/fangohr/octopus-in-docker/issues/22

Reply:
Well spotted, thanks. I tidied up the `Dockerfile-develop` related docs in https://github.com/fangohr/octopus-in-docker/pull/23 . If you are happy with that, we can merge it in here (it is based on this PR).